tests/acceptance/00_basics/04_bundles/main_entry_point.cf

```diff
@@ -0,0 +1,12 @@
+body file control
+{
+      inputs => { "main_entry_point.cf.sub" };
+}
+
+bundle agent __main__
+{
+    methods:
+        "imported" usebundle => imported();
+    reports:
+        "$(this.promise_filename) Pass";
```

Comment:
Shouldn't we print ``Pass`` from in ``reports:`` in the bundle *imported* so that we know it really was processed?